### PR TITLE
ful_SC-19654_add_windows_logs_integration_docs

### PR DESCRIPTION
**Pull request by fulyauluturk**

### CODE REVIEW

File: docs/integration/configuring-windows-agent.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+- System
+- Setup
+
+You can customize which channels the agent collects logs from by following these steps:
+
+1. Navigate to `C:\Program Files\SematextAgent\properties\` and open the `st-agent.yml` file.
+2. Locate the section labeled paths and add or remove channels according to your needs.
+
+```
+paths: ["Security", "Application", "System", "Setup"]
```
Comment (otisg) [inline docs/integration/configuring-windows-agent.md:37]: Are these 4 all available channels or are there more? If the latter, should we list them?

File: docs/integration/windows.md [hunk 2]
```diff
@@ -77,3 +77,58 @@ Moreover, a set of pre-defined [alerts](https://sematext.com/docs/alerts/) is su
...
 In addition to having alerts for these services, you can [create alerts](https://sematext.com/docs/alerts/creating-metrics-alerts/) for any other Windows Service you like, as long as it is available on your Windows machines.
+
+## Logs
+
+Starting from [version 3.7.0](https://sematext.com/docs/agents/sematext-agent/releasenotes/#version-370), [Sematext Agent for Windows](https://sematext.com/docs/agents/sematext-agent/windows-installation/) is capable of collecting Windows Event logs along with metrics related with the status of Windows services.
+
+### Install Sematext Agent
+
+1. Create a Windows Logs [App](https://sematext.com/docs/guide/app-guide/). This will let you install the agent and control access to your logs data.
+2. Install [Sematext Agent for Windows](https://sematext.com/docs/agents/sematext-agent/windows-installation/) according to the instructions displayed in the UI
```
Comment (otisg) [inline docs/integration/windows.md:88]: Install [Sematext Agent for Windows](https://sematext.com/docs/agents/sematext-agent/windows-installation/) according ... ==> Install [Sematext Agent for Windows](https://sematext.com/docs/agents/sematext-agent/windows-installation/) on each Windows host you want to monitor according .... ?

File: docs/integration/windows.md [hunk 3]
```diff
@@ -77,3 +77,58 @@ Moreover, a set of pre-defined [alerts](https://sematext.com/docs/alerts/) is su
...
+3. Once you install the agent, you will start seeing logs from your Windows environment in your [App](https://sematext.com/docs/guide/app-guide/) within a few minutes.
```
Comment (otisg) [inline docs/integration/windows.md:89]: Now quite from your Windows environment, but only from the Windows server(s) where you installed STAW. Saying "environment" may be misleading.

File: docs/integration/windows.md [hunk 4]
```diff
@@ -77,3 +77,58 @@ Moreover, a set of pre-defined [alerts](https://sematext.com/docs/alerts/) is su
...
+- **Scheduled Task Created**: Tracks the creation of new scheduled tasks, which are automated jobs that run at set times. It helps track automation and ensure that scheduled tasks are legitimate.
+- **Logons (Successful, Failed, Logon with Explicit Credentials)**: These logs capture user logon attempts, including successful logins, failed attempts, and those using specific credentials. They help monitor user access and detect potential login issues or unauthorized access attempts.
+- **Remote Desktop Connections**: This log shows when a user connects to the system using Remote Desktop. It helps track remote access and ensure that connections are authorized and secure.
+
+By default, certain critical security events, such as file auditing, audit policy changes and registry changes, are not logged by Windows Servers.To make sure all necessary security events are tracked, these need to be manually enabled in the server’s auditing settings. See [How to Enable Security Logs](https://sematext.com/docs/integration/how-to-enable-windows-security-logs/) for more information.
```
Comment (otisg) [inline docs/integration/windows.md:127]: Servers.To  ==> Servers. To

**Review by otisg — COMMENTED** [review]
Minor comments for text. Will send separate message re screens.

File: docs/integration/configuring-windows-agent.md (hunk 1 above)
Comment (fulyauluturk) [reply docs/integration/configuring-windows-agent.md:37]: These are the default ones. But people can add custom ones based on their configuration.

File: docs/integration/windows.md (hunk 3 above)
Comment (fulyauluturk) [reply docs/integration/windows.md]: Changed it to operating system

File: docs/integration/configuring-windows-agent.md (hunk 1 above)
Comment (otisg) [reply docs/integration/configuring-windows-agent.md:37]: How does one know which other ones are valid strings? What happens if one types FooBar here?

File: docs/integration/windows.md [hunk 5]
```diff
@@ -85,8 +85,8 @@ Starting from [version 3.7.0](https://sematext.com/docs/agents/sematext-agent/re
 ### Install Sematext Agent
 
 1. Create a Windows Logs [App](https://sematext.com/docs/guide/app-guide/). This will let you install the agent and control access to your logs data.
-2. Install [Sematext Agent for Windows](https://sematext.com/docs/agents/sematext-agent/windows-installation/) according to the instructions displayed in the UI
-3. Once you install the agent, you will start seeing logs from your Windows environment in your [App](https://sematext.com/docs/guide/app-guide/) within a few minutes.
+2. Install [Sematext Agent for Windows](https://sematext.com/docs/agents/sematext-agent/windows-installation/) on each host you want according to the instructions displayed in the UI
```
Comment (otisg) [inline docs/integration/windows.md:88]: on each host you want ==> on each host you want to monitor?

**Review by otisg — CHANGES_REQUESTED** [review]

File: docs/integration/configuring-windows-agent.md (hunk 1 above)
Comment (fulyauluturk) [reply docs/integration/configuring-windows-agent.md:37]: The other channels they may want to add will be custom ones, which they've added to Windows Event Viewer later on. They should have the correct string from there. Assuming these are technical users, they will be aware of this. If the channel name is invalid or not found, we won't collect logs from it.

**Review by boratanrikulu — CHANGES_REQUESTED** [review]
Please also update this doc with the new CMD/PowerShell commands: https://github.com/sematext/docs/blob/master/docs/agents/sematext-agent/windows-installation.md

File: docs/integration/configuring-windows-agent.md (hunk 1 above)
Comment (boratanrikulu) [reply docs/integration/configuring-windows-agent.md:37]: If they use "FooBar" it breaks Windows Events collection, we show a log like this one;
```
time="2024-09-19T22:09:50+03:00" level=error msg="skip starting Windows Events Collector: failed to initialize eventlog: subscription of Windows Event Log failed: The specified channel could not be found." source="winevents/events_windows.go:81" visibility=user

```

File: mkdocs.yml [hunk 6]
```diff
@@ -168,6 +168,8 @@ pages:
       - Solr: integration/solr-integration.md
       - Varnish Cache: integration/varnishcache-integration.md
       - Windows: integration/windows.md
+        - Configuring Windows Agent: integration/configuring-windows-agent.md
```
Comment (boratanrikulu) [inline mkdocs.yml:171]: I think this title is too generic

**Comment by boratanrikulu** [comment]
@fulyauluturk I need this doc's link for the release notes.

**Comment by fulyauluturk** [comment]
> @fulyauluturk I need this doc's link for the release notes.

https://sematext.com/docs/integration/configuring-windows-agent/

But note that it is not published yet because Pavao needs to fix some bugs when he comes back next week.

File: mkdocs.yml (hunk 6 above)
Comment (fulyauluturk) [reply mkdocs.yml:171]: Wdym? Any suggestions?

Comment (fulyauluturk) [reply mkdocs.yml]: Updated

File: docs/integration/configuring-windows-agent.md [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+```
+
+3. Save changes and restart the Sematext Agent Service
```
Comment (boratanrikulu) [inline docs/integration/configuring-windows-agent.md:40]: They may ask "how to restart"

File: mkdocs.yml [hunk 8]
```diff
@@ -168,6 +168,8 @@ pages:
       - Solr: integration/solr-integration.md
       - Varnish Cache: integration/varnishcache-integration.md
       - Windows: integration/windows.md
+        - Configuring Logs Collection: integration/configuring-windows-agent.md
```
Comment (boratanrikulu) [inline mkdocs.yml:171]: Please also update this doc with the new CMD/PowerShell commands: https://github.com/sematext/docs/blob/master/docs/agents/sematext-agent/windows-installation.md

File: docs/integration/configuring-windows-agent.md (hunk 7 above)
Comment (boratanrikulu) [reply docs/integration/configuring-windows-agent.md:40]: This doc: https://github.com/sematext/docs/blob/master/docs/agents/sematext-agent/windows-installation.md#how-to-start--stop--restart-sematext-agent

**Pull request merged**